tests/test_metric.py

```diff
@@ -46,3 +49,87 @@ def test_sanitize_metric_name_should_handle_empty_value(self):
         self.assertEqual("", bg_metric.sanitize_metric_name(""))
         self.assertEqual("", bg_metric.sanitize_metric_name("."))
         self.assertEqual("", bg_metric.sanitize_metric_name(".."))
+
+
+class TestMetricMetadata(unittest.TestCase):
+    def test_none(self):
+        """Ensure default parameters."""
+        metadata = bg_metric.MetricMetadata.create()
+        self.assertEqual(metadata.aggregator, bg_metric.Aggregator.average)
+        self.assertEqual(metadata.retention.as_string, "86400*1s:10080*60s")
+        self.assertEqual(metadata.carbon_xfilesfactor, 0.5)
+
+    def test_uniqueness(self):
+        """MetricMetadata.create() should always return the same MetricMetadata instance."""
+        retention1 = bg_metric.Retention.from_string("20*15s:1440*60s:48*3600s")
+        metadata1 = bg_metric.MetricMetadata.create(bg_metric.Aggregator.average, retention1, 0.1)
+        retention2 = bg_metric.Retention.from_string("20*15s:1440*60s:48*3600s")
+        metadata2 = bg_metric.MetricMetadata.create(bg_metric.Aggregator.average, retention2, 0.1)
+        self.assertIs(metadata1, metadata2)
+
+        # the returned instance should't depend on the order of the parameters
+        metadata3 = bg_metric.MetricMetadata.create(
+                                                    carbon_xfilesfactor=0.1,
+                                                    aggregator=bg_metric.Aggregator.average,
```

Comment:
What about extracting variables / constants to show that this is actually the same values than for `metadata1`?